### DISCUSSION TITLE
GNNs for small networks

### DISCUSSION THREAD

**Discussion by antoni4040** [pyg-team/pytorch_geometric#3768] [Q&A] [answered]
Hello. Sorry if this is a bit of a general question, but I don't see a lot of interest in it. I think there is a focus on making GNNs for large graphs, but I'm interested in a problem that has millions of small graphs, less than 20 nodes each. Is there a benchmark dataset for such a problem and are there ideal GNNs in pytorch geometric? I'm looking into the research so far but I'm a bit stuck. Thanks.

**Comment by rusty1s** [comment, marked as answer]
PyG is well-suited for these kinds of problems in my opinion. For example, we have a variety of benchmark examples for all [OGB graph-level tasks](https://ogb.stanford.edu/docs/graphprop/) [here](https://github.com/snap-stanford/ogb/tree/master/examples/graphproppred). I think the [`GIN`](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/mutag_gin.py) network is a great starting place.

**Reply by antoni4040** [reply]
Thanks for the quick reply. Is there some advantage with the GIN model or is it just a bunch of GINConvs as in the example?

**Reply by rusty1s** [reply]
No, they are the same. We refer to GIN as a stack of `GINConv` layers.
